aoc/src/main/scala/aoc2015/Day01_2015.scala

```diff
@@ -0,0 +1,39 @@
+object Day01_2015 extends App:
+  import scala.io.Source
+  import scala.collection.mutable.ListBuffer
+
+  val lines = Source.fromFile("aoc\\src\\resources\\aoc2015\\Day01_2015").getLines.mkString
+
+  def findfloor(current: Int, instructions: String): Int = {
```

Comment:
nothing wrong with a recursive solution - but fyi, you could implement this with a fold over the string, eg.

```
instructions(0)((f,c) =>
  if      (c == '(') f + 1
  else if (c == ')') f - 1
  else               sys.error(s"unknow command: '$c'"))
```